package.json

```diff
@@ -3,13 +3,18 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.27",
```

Comment:
Is this the icon library that Martin is using?

Reply:
The icons we are using match what Martin has, but to my knowledge, he has not listed the library yet.